superset-frontend/stylesheets/less/cosmo/variables.less

```diff
@@ -751,24 +751,6 @@
 @well-bg: @gray-bg; // superset-var
 @well-border: darken(@well-bg, 7%);
 
-// == Badges
-//
-// ##
-
-@badge-color: @lightest; // superset-var
-// ** Linked badge text color on hover
-@badge-link-hover-color: @lightest; // superset-var
-@badge-bg: @brand-primary;
-
-// ** Badge text color in active nav link
-@badge-active-color: @link-color;
-// ** Badge background color in active nav link
-@badge-active-bg: @lightest; // superset-var
-
-@badge-font-weight: bold;
-@badge-line-height: 1;
-@badge-border-radius: 10px;
-
```

Comment:
OMG! So happy to see less LESS.

Reply:
I think it's ok to remove this, but generally I'd keep the theme intact for where FAB is used (the goal is to remove all of the FAB templates, but there are still some CRUD pages that are FAB-powered, and if they have a bootstrap badge they'll won't be themed)

Reply:
Ahh, good catch... I got too excited.

Reply:
Restored. Thanks